src/utils/http-errors.ts

```diff
@@ -11,18 +10,57 @@ export const createHttpError = (statusCode: number, message?: string) => {
   return httpErrors(statusCode ?? 400, message)
 }
 
+export const createError = (
+  title: string,
+  descriptions: string,
+  statusCode: number,
+  message: string
+) => {
```

Comment:
I rather change it to object instead of 4 parameters like the following code
```typescript
export const createError = (error: { title: string, description: string, statusCode: number }) => {
  const { title, description, statusCode } = error

 // blah blah
}
```